docker/ci/Dockerfile

```diff
@@ -20,6 +20,10 @@ RUN apk --no-cache add bash clang git g++ make nodejs openjdk8 python perl \
     && cd $(go env GOPATH)/src/github.com/GoASTScanner/gas \
     && git reset --hard d30c5cde3613e9ba0129febda849e4d4df1d57cd \
     && go install github.com/GoASTScanner/gas
+    && curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly
```

Comment:
This installs Rust into the current user's home directory. I'm guessing the current user here is `root`. Perhaps this needs to be changed to install it as the user wercker uses.